web/src/lib/types/types.ts

```diff
@@ -176,6 +176,7 @@ export interface AnytimeActionTask extends AnytimeAction {
   contentMd: string;
   description?: string;
   searchMetaDataMatch?: string;
+  type: string;
```

Comment:
can this be added under `interface AnytimeAction`, AnytimeActionLicenseReinstatement's type gets set in the anytime action drowdown

Reply:
I'll do that in the next ticket. I'll have to change a number of files to make that work.